common/python/utility/hex_utils.py

```diff
@@ -42,6 +42,21 @@ def is_valid_hex_str(hex_str):
         return False
 
 
+def mrenclave_hex_string(enclave_metadata_file):
+    """
+    Function to extract the MREnclave value from a file
+    generated by generate_mrenclave.sh
```

Comment:
Let the comment not be specific if this function is going into a util afterall. I mean could have been generated by anybody, but the format is what should be taken care of.